src/Directory.Build.props

```diff
@@ -13,8 +13,11 @@
     <Compile Include="$(MSBuildThisFileDirectory)AssemblyInfo.Shared.cs">
       <Link>Properties\AssemblyInfo.Shared.cs</Link>
     </Compile>
-	<AdditionalFiles Include="$(MSBuildThisFileDirectory)vs-threading.MainThreadAssertingMethods.txt">
-	      <Link>Properties\vs-threading.MainThreadAssertingMethods.txt</Link>
```

Comment:
Miscellaneous: the `Link` wasn't required - it was telling VS look in a different place for the file.